packages/wix-ui-core/package.json

```
@@ -5,6 +5,9 @@
   "version": "3.0.0",
   "main": "./dist/src/index.js",
   "module": "./dist/es/src/index.js",
+  "browser": {
+    "./index.st.css": "./index.es.st.css"
+  },
```

```suggestion
  },
  "exports": {
    "./index.st.css": "./index.es.st.css",
  },
```

same as in https://github.com/wix-private/wix-design-systems/pull/9137